services/github/github-tag.service.js

```diff
@@ -84,39 +80,101 @@ module.exports = class GithubTag extends GithubAuthV3Service {
     return { label: 'tag' }
   }
 
-  static render({ usingSemver, version }) {
+  static render({ version, sort }) {
     return {
       message: addv(version),
-      color: usingSemver ? versionColor(version) : 'blue',
+      color: sort === 'semver' ? versionColor(version) : 'blue',
     }
   }
 
-  static transform({ usingSemver, includePre, json }) {
-    const versions = json.map(({ name }) => name)
-    if (usingSemver) {
-      return latest(versions, { pre: includePre })
-    } else {
-      return versions[0]
+  async fetch({ user, repo, sort }) {
+    const limit = sort === 'semver' ? 100 : 1
```

Comment:
Regardless of how many we request here, the query cost is still 1.

100 is an arbitrary number, so open to feedback but I reckon it keeps the response size reasonable. Grabbing the latest 100 releases should be plenty unless you want latest stabe semver and you released version 1.9 and then put out 101 betas for 2.0 ;)

This should be an improvement over what we were doing before using the first 30 in alphabetical order anyway, and fixes issues like #3080 :)

Reply:
The `per_page` query param for the V3 API had a max value of 100, so that feels like a natural choice to me